tree/30_generic_methods/file_check_block_device.cf

```diff
@@ -24,8 +24,8 @@
 # 
 # @class_prefix file_check_block_device
 # @class_parameter file_name
-# This bundle will define a class file_check_block_device_${file_name}_{ok, reached, kept} if the
-# file is a block_device, or file_check_block_device_${file_name}_{not_ok, reached, not_kept, failed} if
+# This bundle will define a class `file_check_block_device_${file_name}_{ok, reached, kept}` if the
```

Comment:
Indeed, the PR that added these methods was started before the changes in ncf docs, and merged after. I will make a separate ticket for these.